modules/gateway/gateway.go

```diff
@@ -81,15 +81,7 @@ import (
 // have previously been outbound peers, as it is less likely that those have
 // been manipulated.
 //
-// TODO: When peers connect to eachother, and when they add nodes to the node
-// list, there is no verification that the peers are running on the same Sia
-// network, something that will be problematic if we set up a large testnet.
-// It's already problematic, and currently only solved by using a different set
-// of bootstrap nodes. If there is any cross-polination (which an attacker
-// could do pretty easily), the gateways will not clean up over time, which
-// will degrade the quality of the flood network as the two networks will
-// continously flood eachother with irrelevant information. Additionally, there
-// is no public key exhcange, so communications cannot be effectively encrypted
+// TODO: There is no public key exhcange, so communications cannot be effectively encrypted
```

Comment:
Oh nice, I quite like that comment, I will recheck my comment adds/mods. I also like to keep them short, but am never too exact about the 80, as I usually just guess. I will make sure that in your codebase it all first in 80 chars. Does this also count for other code, or just comments?